f8a_worker/storages/postgres.py

```diff
@@ -286,9 +289,13 @@ def validate_utf_json(result):
     :param result: dict, Dictionary having details of task
     return: a dictionary received as input having non UTF characters removed if present
     """
-    if 'details' in result and isinstance(result['details'], list) \
-            and len(result['details']) > 0 and 'description' in result['details'][0]:
-        result['details'][0]['description'] = ''.join(
-            x for x in result['details'][0]['description']
-            if x in string.printable)
-    return result
+    try:
+        if result is not None and 'details' in result and isinstance(result['details'], list) \
+                and len(result['details']) > 0 and 'description' in result['details'][0]:
```

Comment:
I'm finding little difficulty in following the purpose of the condition. After so many criteria, we are checking whether the `details` field has printable chars. 

If the criteria fails we are returning the result as such. Does it mean only specific worker type sets `details` field? If that is the case, can't we validate the `details` in that particular worker itself?

Reply:
Yes @arajkumar, only some places only description is present, as this is the common place so I have done it here.
Allow me some time to list down those places and see if I can write this checks there.